src/server/pkg/testutil/auth.go

```diff
@@ -100,20 +92,16 @@ func getPachClientInternal(tb testing.TB, subject string) *client.APIClient {
 	if subject == "" {
 		return resultClient // anonymous client
 	}
+	if subject == AdminUser {
+		resultClient.SetAuthToken(RootToken)
+		return resultClient
+	}
+
 	if token, ok := tokenMap[subject]; ok {
```

Comment:
Does this code make sense to keep around if `UpdateAuthToken` is going away? IIUC, there isn't a way to put tokens into `tokenMap` anymore, which means this code won't run anymore either

Reply:
I'm going to refactor this further in a follow-on, once we make the "root is always an admin" change most of this file can go away because we just use the root client to make a token for whoever.